server/api/src/rsGeneralAdmin.cpp

```diff
@@ -52,6 +53,63 @@ namespace
         return std::find_if(b, e, [](unsigned char _ch) { return ::isspace(_ch); }) != e;
     } // contains_whitespace
 
+     auto throw_if_inappropriate_rodsadmin_downgrade(
+        RsComm& rsComm,
+        const std::string_view& _new_value,
```

Comment:
Consider renaming this parameter. `_new_value` is a little too general.

Perhaps, `_new_user_type`?